### PR TITLE
refactor(Chunk): use only delete method instead of `has`,`delete` chain

### PR DESCRIPTION
**Pull request by vankop**
**What kind of change does this PR introduce?**

refactoring

**Did you add tests for your changes?**

no

**Does this PR introduce a breaking change?**

yes, removeGroup and addGroup have no longer return values

**What needs to be documented once your changes are merged?**

nothing

### CODE REVIEW

File: lib/Chunk.js [hunk 1]
```diff
@@ -438,9 +438,7 @@ class Chunk {
 	 * @returns {boolean} returns true if chunk does exist in chunkGroup and is removed
 	 */
 	removeGroup(chunkGroup) {
```
Comment (sokra) [inline lib/Chunk.js]: addGroup above need to be changed in similar way.

Comment (vankop) [reply lib/Chunk.js]: Unfortunately, it works other way https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/add#Return_value

Comment (vankop) [reply lib/Chunk.js]: We can change to `add`, `has` chain, don't know will this impact performance.

Comment (sokra) [reply lib/Chunk.js]: ah I see what you did.

You can remove the return values from `removeGroup` and `addGroup` and make them return void methods. The result is never used in the codebase.

Comment (vankop) [reply lib/Chunk.js]: Done

File: lib/Chunk.js [hunk 2]
```diff
@@ -425,22 +425,18 @@ class Chunk {
 
 	/**
 	 * @param {ChunkGroup} chunkGroup the chunkGroup the chunk is being added
-	 * @returns {boolean} returns true if chunk is not a part of chunkGroup and is added successfully
+	 * @returns {void}
 	 */
 	addGroup(chunkGroup) {
-		if (this._groups.has(chunkGroup)) return false;
 		this._groups.add(chunkGroup);
-		return true;
```
Comment (smelukov) [inline lib/Chunk.js]: seems like a breaking change for webpack 4
e.g. some plugins based on this result

Comment (sokra) [reply lib/Chunk.js]: yes

**Comment by sokra** [comment]
Thanks

**Pull request merged**